mbtiles/src/mbtiles.rs

```diff
@@ -39,6 +39,60 @@ pub struct Metadata {
     pub json: Option<JSONValue>,
 }
 
+#[derive(Clone, Debug, PartialEq, Serialize)]
+pub struct LevelDetail {
+    pub zoom: String,
+    pub count: u32,
+    pub smallest: f32,
+    pub largest: f32,
+    pub average: f32,
+    pub bounding_box: [f32; 4],
+}
+
+#[derive(Clone, Debug, PartialEq, Serialize)]
+pub struct Statistics {
+    pub file_path: String,
+    pub file_size: f64,
+    pub schema: String,
```

Comment:
I think it is better to keep the value as original until serialization.  When serializing, simply use the Display variant (i.e. just print it as is, and let MbtType format itself)

```suggestion
    pub schema: MbtType,
```